.buildkite/shared-pipeline-vars

```diff
@@ -0,0 +1,9 @@
+#!/bin/sh
+
+# This file is `source`'d before calling `buildkite-agent pipeline upload`, and can be used
+# to set up some variables that will be interpolated in the `.yml` pipeline before uploading it.
+
+export IMAGE_ID="xcode-26.0-beta"
```

Comment:
I think `26.0-beta` is fine for this project: it's not meant to be released very soon, but the project can already align with the newest tooling.

Reply:
I'd tend to agree, but I'd also let this kind of decision to be validate by the dev team first just to be sure :) /cc @pinarol @andrewdmontgomery

Reply:
Sure 👍 addressed via 37bad2449df386dcc29ee9cc06f30543f37f0681